Commons/src/main/java/eu/neclab/ngsildbroker/commons/storage/StorageReaderDAO.java

```diff
@@ -54,20 +57,15 @@ public String findDataBaseNameByTenantId(String tenantidvalue) {
 		if (tenantidvalue == null)
 			return null;
 		try {
-			synchronized (readerJdbcTemplate) {
-				readerJdbcTemplate = new JdbcTemplate(masterDataSource);
-			}
+
 			// String databasename="ngbcsource2";
 			// SELECT EXISTS(SELECT datname FROM pg_database WHERE datname = 'tenant2');
 			String sql = "SELECT database_name FROM tenant WHERE tenant_id = ?";
```

Comment:
[major]only suggestion, No need to write the select query operation here so we can compare "tenantidValue"with append string with prefix database name "ngb" like as ngb+tenantidValue..

Reply:
Done